sale_service_project/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Sale Service Project
+====================
+
+This module was written to extends the functionality of sale workflow
+and allow you to create projects and tasks with materials and works
+from sale lines.
+You can invoice from sale order or analytic lines.
+
+Configuration
+=============
+
+To use this module, you need to:
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Create a service product and check 'Create Task Automatically'.
+2. Now you can add task works and materials for this product.
```

Comment:
These usage instructions definitely need more precision:

- where to go to make each action?
- what new things you will see in the product, sale order, sale order line, project task...?
- what do they mean?
- what would happen if you add works for tasks in the product? Do the hours mean they are the expected time for task, or are fulfilled time for task works?

I had to guess most of those, so please explain usage better.